Formula 1/ViewControllers/RootTabBarController.swift

```diff
@@ -18,6 +18,9 @@ class RootTabBarController: UITabBarController {
         let secondVC =  UINavigationController(rootViewController: ConstructorsCollectionViewController(nibName: nil, bundle: nil))
         secondVC.tabBarItem.title = "Constructors"
         
-        viewControllers = [firstVC, secondVC]
+        let thirdVC =  UINavigationController(rootViewController: CircuitsCollectionViewController())
```

Comment:
you may want to keep a reference to this navcontroller and then maybe have this tabBarController be the VC in charge of navigations. So that tabs use a delegate to tell the tabBarController to navigate somewhere.